addons/survey/static/src/scss/survey_templates_form.scss

```diff
@@ -458,4 +458,51 @@ _::-webkit-full-page-media, _:future, :root .o_survey_wrap {
             color: $gray-600;
         }
     }
+    .o_survey_choice_image img {
+        cursor: default;
+        &:hover {
+            box-shadow: none;
+        }
+    }
+}
+
+/**********************************************************
+            Image (Zoom widget) Specific Style
+ **********************************************************/
+
+.o_survey_choice_image_modal {
+    .modal-dialog {
+        height: 100%;
+        max-width: 100%;
+        background-color: rgba(0,0,0,0.8);
+        padding-bottom: 0px;
+        padding-top: 0px;
+    }
+    .o_survey_slide {
+        display: flex;
+        align-items: center;
```

Comment:
you can use bootstrap classes instead for these 4.

Reply:
Actually the last two do not seem necessary at all.